notebooks/ltx-video/ltx-video.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from ov_ltx_video_helper import convert_text_encoder, convert_transformer, convert_vae_decoder
</pre><p>could you please add comment with possibility to revise conversion code?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>added</p>